### DISCUSSION TITLE
Questions in different languages

### DISCUSSION THREAD

**Discussion by nicogithub22** [globaleaks/globaleaks-whistleblowing-software#3396] [Q&A]
### What version of GlobaLeaks are you using?

4.11.0

### What operating system(s) are you seeing the problem on?

Windows

### Describe the issue

Hello, is it possible to offer the questions in different languages or do I have to open an extra channel for this?

**Comment by evilaliv3** [comment]
Hello @nicogithub22 , yes this is possible and you do not need a different channel.

Just enable the languages that you need, switch to the specific language and configure the translations of the same questionnaire.

Basically when you operate using a specific language you are operating the configuration in that language.

Please let us know if this is clear or if you have any doubt.

**Reply by nicogithub22** [reply]
> Hello @nicogithub22 , yes this is possible and you do not need a different channel.
> 
> Just enable the languages that you need, switch to the specific language and configure the translations of the same questionnaire.
> 
> Basically when you operate using a specific language you are operating the configuration in that language.
> 
> Please let us know if this is clear or if you have any doubt.

I have checked the translation but not the original. Now I have seen that no matter which language I select as a user, the language of the questions remains the same.
My steps:
1. log in with an admin account
2. changed the language in the top right corner 
3. translated question catalogues
4. saved everything 
Doesn't work like this, what am I doing wrong?

**Comment by nicogithub22** [comment]
Thank you, thats works. Is it also possible to change the project name dynamically?
(I tried the same way as with the questions, but it did not work.)

**Comment by evilaliv3** [comment]
No actually, the Project name is designed to be the same in every language as this is what typically projects do for branding purposes.
You have the possibility to configure a title for the home page and that title is instead subject to internationalization.

**Comment by evilaliv3** [comment]
Yes It is correct.

> Message ID: <globaleaks/GlobaLeaks/repo-discussions/3396/comments/5406420@
> github.com>
>

Do you have a public link that I could check?

**Comment by nicogithub22** [comment]
It didn't work with the questionnaires I created first, but it worked with new ones. Problem solved for now. THX
